core/trino-parser/src/main/java/io/trino/sql/tree/CreateTable.java

```diff
@@ -27,25 +27,39 @@
 {
     private final QualifiedName name;
     private final List<TableElement> elements;
+    private final boolean replace;
     private final boolean notExists;
     private final List<Property> properties;
     private final Optional<String> comment;
 
+    @Deprecated
     public CreateTable(QualifiedName name, List<TableElement> elements, boolean notExists, List<Property> properties, Optional<String> comment)
     {
-        this(Optional.empty(), name, elements, notExists, properties, comment);
+        this(Optional.empty(), name, elements, false, notExists, properties, comment);
     }
 
+    public CreateTable(QualifiedName name, List<TableElement> elements, boolean replace, boolean notExists, List<Property> properties, Optional<String> comment)
+    {
+        this(Optional.empty(), name, elements, replace, notExists, properties, comment);
+    }
+
+    @Deprecated
```

Comment:
Remove the old constructor. It should only be called by `AstBuilder` (and tests).